README.rst

```diff
@@ -516,6 +516,7 @@ Please submit a pull request if you wish to include your project on the list:
   `@matthiaszimmermann <https://github.com/matthiaszimmermann>`_
 - `Trust Ethereum Wallet <https://github.com/TrustWallet/trust-wallet-android>`_
 - `Presto Ethereum <https://github.com/xiaoyao1991/presto-ethereum>`_
+- [Kundera-Ethereum data importer and sync utility] (https://github.com/impetus-opensource/Kundera/tree/trunk/src/kundera-ethereum) by [@impetus-opensource] (https://github.com/impetus-opensource)
```

Comment:
Please can you format this using ReStructuredText, not Markdown

Reply:
sure